test/odbc/psqlodbc/bigint.cpp

```diff
@@ -0,0 +1,411 @@
+  for (int i = 0; i < NUM_OF_DATA; i++) {
+    insertString += comma + "(" + std::to_string(i) + ",\'" + INSERTED_DATA[i] + "\')";
+    comma = ",";
+  }
+
+  const vector<string> OPERATIONS_QUERY = {
+    "MIN(" + COL2_NAME + ")",
+    "MAX(" + COL2_NAME + ")",
+    "SUM(" + COL2_NAME + ")",
+    "AVG(" + COL2_NAME + ")"
+  };
+  const int NUM_OF_OPERATIONS = OPERATIONS_QUERY.size();
+
+  const vector<long> expectedLen(NUM_OF_OPERATIONS, BIGINT_BYTES_EXPECTED);
+
+  // initialization of expected_results
+  vector<long long int> expected_results = {};
+
+  long long int curr = StringToBigInt(INSERTED_DATA[0]);
+  long long int min_expected = curr, max_expected = curr, sum = curr;
+
+  for (int i = 1; i < NUM_OF_DATA; i++) {
+    curr = StringToBigInt(INSERTED_DATA[i]);
+    sum += curr;
+
+    min_expected = std::min(curr, min_expected);
+    max_expected = std::max(curr, max_expected);
+  }
+  expected_results.push_back(min_expected);
+  expected_results.push_back(max_expected);
+  expected_results.push_back(sum);
+  expected_results.push_back(sum / NUM_OF_DATA);
+
+  // Create a vector of length NUM_OF_OPERATIONS with dummy value of -1 to store column results
+  vector<long long int> col_results(NUM_OF_OPERATIONS, -1);
+  
+  createTable(ServerType::PSQL, TABLE_NAME, TABLE_COLUMNS);
+  insertValuesInTable(ServerType::PSQL, TABLE_NAME, insertString, NUM_OF_DATA);
+  testComparisonFunctions(ServerType::PSQL, TABLE_NAME, SQL_C_SBIGINT, col_results, bufferLen, OPERATIONS_QUERY, expected_results, expectedLen);
+  dropObject(ServerType::PSQL, "TABLE", TABLE_NAME);
+}
+
+TEST_F(PSQL_DataTypes_BigInt, View_Creation) {
+  const string VIEW_QUERY = "SELECT * FROM " + TABLE_NAME;
+
+  long long int data;
+  const int bufferLen = 0;
+
+  const vector<string> INSERTED_DATA = {
+    "9223372036854775807",
+    "123456789",
+    "-9223372036854775808"
+  };
+  
+  const vector<long long int> EXPECTED_DATA = {
+    StringToBigInt("9223372036854775807"),
+    StringToBigInt("123456789"),
+    StringToBigInt("-9223372036854775808")
+  };
+
+  const vector<long> expectedLen(EXPECTED_DATA.size(), BIGINT_BYTES_EXPECTED);
+
+  createTable(ServerType::PSQL, TABLE_NAME, TABLE_COLUMNS);
+  testInsertionSuccess(ServerType::PSQL, TABLE_NAME, COL1_NAME, SQL_C_SBIGINT, data, bufferLen, INSERTED_DATA, EXPECTED_DATA, expectedLen);
+
+  createView(ServerType::PSQL, VIEW_NAME, VIEW_QUERY);
+  verifyValuesInObject(ServerType::PSQL, VIEW_NAME, COL1_NAME, SQL_C_SBIGINT, data, bufferLen, INSERTED_DATA, EXPECTED_DATA, expectedLen);
+  
+  dropObject(ServerType::PSQL, "VIEW", VIEW_NAME);
+  dropObject(ServerType::PSQL, "TABLE", TABLE_NAME);
+}
+
+TEST_F(PSQL_DataTypes_BigInt, Table_Unique_Constraints) {
+  const vector<pair<string, string>> TABLE_COLUMNS = {
+    {COL1_NAME, "INT PRIMARY KEY"},
+    {COL2_NAME, DATATYPE_NAME + " UNIQUE"}
+  };
+
+  const vector<string> UNIQUE_COLUMNS = {COL2_NAME};
+
+  string tableConstraints = createTableConstraint("UNIQUE", UNIQUE_COLUMNS);
```

Comment:
When creating constraints for the table, which one is used or are they duplicated
The section in `TABLE_COLUMNS` or `tableConstraints`

Reply:
Good catch. Fixed, removed the "UNIQUE" keyword in `TABLE_COLUMNS`.